airgun/widgets.py

```diff
@@ -2188,3 +2190,43 @@ def fill(self, item):
     def read(self):
         """read column updated value"""
         return self.column_value.read()
+
+
+class AuthSourceAggregateCard(AggregateStatusCard):
+    """This is a customizable card widget which is has the title, count and kebab widget
+
+    Example html representation::
+
+        <div class="card-pf-body">
+            <div class="dropdown pull-right dropdown-kebab-pf">
+                <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropupKebabRight2">
+                    <li><a href="/auth_source_externals/3/edit">Edit</a></li>
+                </ul>
+            </div>
+            <h2 class="card-pf-title text-center">
+                    External
+            </h2>
+            <div class="card-pf-items text-center">
+                <div class="card-pf-item">
+                    <span class="pficon pficon-users"></span>
+                    <span class="card-pf-item-text"><a href="">10</a></span>
+                </div>
+            </div>
+        </div>
+    """
+    select_kebab = Kebab(locator="./div[contains(@class, 'dropdown-kebab-pf')]")
```

Comment:
What is your plan for that? Is this element necessary?
It seems you don't interact with it anywhere so far.

Reply:
I just kept it, for now, I will add more action in case necessary. It is mainly used for the creation or edit. I will add Github issue for this.